css/style.css

```diff
@@ -42,12 +39,120 @@ body {
   width: 100%;
   margin: auto;
   height: 500px;
+  background: url(../images/header_bg.jpg) no-repeat center top rgb(134, 141, 157);
+  background-size: cover;
+  }
+
+  .container2 {
+    width: 100%;
+    margin: auto;
+    height: 300px;
```

Comment:
Try to avoid setting explicit `height`s on elements that contain text. If a user were to look at this on a smaller screen, the text would end up overflowing the container. If you need a height, try using `min-height`. this way the text can grow beyond whatever height you set if it needs to.

This also applies to `.container` above.